### PR TITLE
Minor fixes and improvements

### PR DESCRIPTION
**Pull request by karthiknadig**
1. RTVS daemon installation uses the snakeoil pem and key as the default self-signed SSL certificate.
2. Fix and updates to container build.

### CODE REVIEW

File: src/Containers/Impl/Docker/LocalDockerService.cs [hunk 1]
```diff
@@ -149,13 +150,24 @@ public Task<string> RepositoryLogoutAsync(RepositoryCredentials auth, Cancellati
 
             var output = await process.StandardOutput.ReadToEndAsync();
             var error = await process.StandardError.ReadToEndAsync();
-            if (!string.IsNullOrEmpty(error)) {
+            if (!string.IsNullOrEmpty(error)  && !IsSecurityWarning(error)) {
                 _outputLogWriter?.Write(MessageCategory.Error, error);
                 throw new ContainerException(error);
             }
 
+            if (!process.HasExited) {
```
Comment (AlexanderSher) [inline src/Containers/Impl/Docker/LocalDockerService.cs]: How is this possible?

Comment (karthiknadig) [reply src/Containers/Impl/Docker/LocalDockerService.cs]: I missed removing this code. I had added this while testing the "it leaves behind a window" problem.

**Comment by AlexanderSher** [comment]
@int19h , can you please take a look?

**Comment by karthiknadig** [comment]
@int19h This is the same thing I discussed with you over email. Basically using the snakeoil pem and key to generate the PFX. This removes the need for the user to create their own SSL cert.

**Pull request merged**